bundles/org.openhab.io.homekit/README.md

```diff
@@ -2,6 +2,9 @@
 
 This is an add-on that exposes your openHAB system as a bridge over the HomeKit protocol.
 
+This integration **exports** openHAB Items to a HomeKit controller.
+Do not confuse this with the other HomeKit **binding** (https://www.openhab.org/addons/bindings/homekit/) which **imports** data from HomeKit accessories.
```

Comment:
```suggestion
Do not confuse this with the HomeKit **binding** (https://www.openhab.org/addons/bindings/homekit/) which **imports** data from HomeKit accessories so that you can control them directly from openHAB.
```